CHANGELOG.md

```diff
@@ -1,8 +1,5 @@
-<<<<<<< HEAD
```

Comment:
should be cleaned up once you merge in `main` since this has already been fixed in `main`